rebar.config

```diff
@@ -1,8 +1,7 @@
-{require_otp_vsn, "R15|R16|(17.*)|(18.*)"}.
 {erl_opts, [debug_info,
             {parse_transform, lager_transform}]}.
 
 {deps, [
-        {lager, ".*", {git, "https://github.com/basho/lager.git", {tag, "2.1.0"}}}
+        lager
```

Comment:
Could also be `{lager, "3.5.0"}` if you want to pin it to a release. Otherwise, you get the latest.